src/interface/src/app/plan/plan-table/plan-table.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="root-container">
+  <h1>My Plans</h1>
+
+  <div class="button-row">
+    <button mat-raised-button color="primary" (click)="create()">
+      <mat-icon>add_box</mat-icon>
+      CREATE
+    </button>
+  </div>
+
+  <mat-table [dataSource]="datasource" matSort>
+    <!-- Name Column -->
+    <ng-container matColumnDef="name">
+      <mat-header-cell *matHeaderCellDef mat-sort-header>
+        Name
+      </mat-header-cell>
+      <mat-cell *matCellDef="let element">
+        <a href="/plan/{{ element.id }}">{{element.name}}</a>
```

Comment:
Nit: can the extra spaces around "element.id" be removed to be consistent with the rest of the references in the file?

Reply:
Done!